tests/ssg_test_suite/common.py

```diff
@@ -18,6 +18,38 @@
     "-o", "UserKnownHostsFile=/dev/null",
 )
 
+# Keys are strings used in test scenario header in 'platform' list
+# CentOS CPEs are added to allow testing RHEL profiles on CentOS VMs
+TEST_PLATFORMS = {
+    "Red Hat Enterprise Linux 6": [
+        "cpe:/o:redhat:enterprise_linux:6",
+        "cpe:/o:centos:centos:6"
+    ],
+    "Red Hat Enterprise Linux 7": [
+        "cpe:/o:redhat:enterprise_linux:7",
+        "cpe:/o:centos:centos:7"
+    ],
+    "Red Hat Enterprise Linux 8": [
+        "cpe:/o:redhat:enterprise_linux:8",
+        "cpe:/o:centos:centos:8"
+    ],
+    "Fedora": [
+        "cpe:/o:fedoraproject:fedora:28",
+        "cpe:/o:fedoraproject:fedora:29",
+        "cpe:/o:fedoraproject:fedora:30"
+    ],
+}
+
+# Keys are strings used in test scenario header in 'platform' list
+TEST_MULTI_PLATFORMS = {
```

Comment:
In `ssg/products.py` we already maintain a mapping of platforms to names.
I'd rather avoid maintaining a separate mapping, and extend that functionality to map from platforms to cpes.

Reply:
@yuumasato  Yes, I like the idea to have it defined at single place.

There are multiple problems with that:
* The CPEs for each product are defined in the product-specific `product.yml`. We probably shouldn't parse these files in the test suite. It would be better to change the content build system to source it from a single place as well.
* The CPEs for derivatives (CentOS, Scientific Linux) are not defined in `product.yml` but `ssg/constants.py`.
* The code in `ssg/products.py` maps shortened names of products (eg. `rhel7`) to official names, eg. when you build `rhel7` product you need to know that you  should also include remediations marked as one of  `['multi_platform_rhel','multi_platform_all', 'Red Hat Enterprise Linux 7']`.  But we need kind of opposite mapping - eg. when we specify `multi_platform_rhel` we would like to expand it to `['rhel6','rhel7']` and then convert it a list of CPEs.

I would like to have all the mappings defined at a single point. But I'm not sure if it's not a scope creep. Should it be done in a separate PR?

Reply:
> * The CPEs for each product are defined in the product-specific `product.yml`. We probably shouldn't parse these files in the test suite. It would be better to change the content build system to source it from a single place as well.

I'm not sure it would be better, for someone writing content, or adding a new product, it wouldn't be straight forward to go into `ssg/constants.py` and add the mappings there.

I'm okay to go take this path for now, but in the end, I think all data specific for the product should be in its `product.yml`, for example, the product tag name, full name, and cpes.
And the build system would know what products are part of which `multi_platform_` label, and it gathers the data and builds the mappings.

This mapping of product could even be extended to something else, something not tied to the product name, but feature based.  This could be a "label" defined in the `product.yml`, that would aggregate rules, to products that provide the feature.
Some labels I can imagine:`crypto_policy`, `fips`, `grub_legacy`, `grub2`.

Reply:
@jan-cerny yes. In the future we could.